templates/accuracy_notice_email.html

```diff
@@ -0,0 +1,31 @@
+<p>
+    [Why me?] You are receiving this notification because you are listed as an owner of the following ChromeStatus feature entry:
+</p>
+</br>
+<p>
+    <strong>{{feature_name}}</strong>
+</p> 
+</br>
+<p>
+    [Why now?] Your feature is slated to launch soon:
+</p>
+</br>
+<p>
+    <strong>m{{milestone}}</strong>
+</p>
+</br>
+<p>
+    [Background/motivation] Your feature entry is an important resource for
+    cross functional teams that help drive adoption of new features and
+    enterprise IT admins who might be affected by web platform changes.
+</p>
+</br>
+<p>
+    [Call to action] We need to know whether your plans are changing or staying
+    the same. <strong>Please click the link below to update and confirm key
+    fields of your feature entry.</strong>
+</p>
+</br>
+<b><a href="https://chromestatus.com/guide/verify_accuracy/{{id}}">
```

Comment:
@jrobbins Does this seem like a good choice for the page to verify feature data accuracy?

Reply:
Sure, it says what it does.  I like it.